web/src/editor.js

```diff
@@ -51,12 +51,12 @@ class Editor extends Component {
 
       keyService.bindings.forEach(({ keystroke }) => {
         const cpIndex = keymap.findIndex(x => x.keys === keystroke.vimKey);
-        if (~cpIndex) {
+        if (cpIndex !== -1) {
```

Comment:
I'm not 100% certain on the intent here, but it seems like this has the same behavior as `~cpIndex`. `~cpIndex` will be truthy unless `cpIndex` is `-1`.